src/components/LoginPage/LoginPageComponent.js

```diff
@@ -91,7 +91,7 @@ export class LoginPageComponent extends Component {
             <H3><span>Neural</span>Notes is open source and free</H3>
             <GoogleLoginButton type="button" onClick={authHandler}>
               <img src={iconGoogle} />
-              <span>Sing up with Google</span>
+              <span>Sign up with Google</span>
```

Comment:
This is not related to the issue. It shouldn't be changed in the "Convert tree DS to graph" PR.

Otherwise if we merge this PR and then revert it, this typo will be committed again.

Reply:
reverted